src/transformers/models/opt/modeling_opt.py

```diff
@@ -836,7 +836,7 @@ def __init__(self, config):
         self.model = OPTModel(config)
 
         # the lm_head weight is automatically tied to the embed tokens weight
-        self.lm_head = nn.Linear(config.hidden_size, config.vocab_size, bias=False)
+        self.lm_head = nn.Linear(config.word_embed_proj_dim, config.vocab_size, bias=False)
```

Comment:
Yes, this looks correct to me! So far it never raised an error because the weights were tied for `from_pretrained(...)`